toolium/pageelements/input_text_page_element.py

```diff
@@ -26,7 +26,7 @@ def text(self):
 
         :returns: element text value
         """
-        if self.driver_wrapper.is_web_test():
+        if self.driver_wrapper.is_web_test() or self.webview:
```

Comment:
Should this change apply also to `def text(self, value)` method?

Reply:
No, for the setter is working fine